radio/src/sdcard.cpp

```diff
@@ -141,3 +141,47 @@ bool listSdFiles(const char *path, const char *extension, const uint8_t maxlen,
 
   return s_menu_count;
 }
+
+const char *fileCopy(const char *filename, const char *srcDir, const char *destDir)
+{
+  FIL srcFile;
+  FIL dstFile;
+  char buf[256];
+  UINT read = sizeof(buf);
+  UINT written = sizeof(buf);
+
+  char path[2*CLIPBOARD_PATH_LEN+1];
+  char *tmp = strAppend(path, srcDir, CLIPBOARD_PATH_LEN);
+  *tmp++ = '/';
+  strAppend(tmp, filename, CLIPBOARD_PATH_LEN);
```

Comment:
Obviously yes, I think I wanted to put a break or I wouldn't have written dest++ there!